scripts/set_gradle_proxy.sh

```diff
@@ -32,12 +32,14 @@ writeProxyStringToGradleProps () {
         # Strip authentication info.
         HOST=${HOST#$AUTH@}
         # Extract the user.
-        local USER=${AUTH%%:*}
+        local PROXY_USER=${AUTH%%:*}
         # Extract the password.
-        local PASSWORD=${AUTH#*:}
+        local PROXY_PASSWORD=${AUTH#*:}
```

Comment:
I'm curious: As these are local variables, I though there could be no conflict with a global `USER` environment variable. In the message for this commit, could you please go a bit into the details of the problem you were facing with the original code?

Reply:
Commit message nits:
- In the body, just start the second sentence in the first line.
- http -> HTTPS
- gradle -> Gradle